### PR TITLE
Rebuild data models for Builds, add simple Display for example build

### PR DESCRIPTION
**Pull request by ahubers**
## Motivation
I wanted to change builds and I wanted to restructure the App + flesh out the ducks for creating and getting a build order. 

This PR is a bit bigger than I usually like to make them but it involves some restructuring + the requisite work needed for `/build/:id` route to work.

### CODE REVIEW

**Comment by tstirrat15** [comment]
LGTM

File: apps/common/feathers/client.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const feathers = require('@feathersjs/feathers');
+const rest = require('@feathersjs/rest-client');
+
+const app = feathers();
+
+// Connect to the same as the browser URL (only in the browser)
+const restClient = rest();
+
+// eslint-disable-next-line no-undef
+if (window && window.fetch) {
```
Comment (tstirrat15) [inline apps/common/feathers/client.js:10]: This'd be a good place to use a [fetch polyfill](https://github.com/github/fetch). It'd also make the import nicer.

**Review by tstirrat15 — COMMENTED** [review]
See comment

File: apps/common/feathers/client.js (hunk 1 above)
Comment (ahubers) [reply apps/common/feathers/client.js:10]: yar -- i ignored here because this is only client-side code

**Pull request merged**